File: C#/Lv_0/배열의 유사도.cs
using System;

public class 배열의_유사도 {
    public int solution(string[] s1, string[] s2) {
        int answer = 0;
        for(int i=0; i<s1.Length; i++)
            for(int j=0; j<s2.Length; j++)
                if(string.Compare(s1[i], s2[j], false) == 0) answer++;
        return answer;
    }
}

File: JavaScript/Lv_0/배열의 유사도.js
function solution(s1, s2) {
    var answer = 0;
    for(let i=0; i<s1.length; i++)
        for(let j=0; j<s2.length; j++)
            if(s1[i] === s2[j]) answer++;
    return answer;
}

// filter를 사용하면 더 간단하다.
// function solution(s1, s2) {
//     return s1.filter(x => s2.includes(x)).length;
// }